### PR TITLE
Rename Mail to Email

### PR DESCRIPTION
**Pull request by docelic**

### CODE REVIEW

File: email.c [hunk 1]
```diff
@@ -2,13 +2,13 @@
 #include <stdlib.h>
 #include <curl/curl.h>
 
-#include <urweb.h>
+#include <urweb/urweb.h>
```
Comment (mdempsky) [inline email.c:5]: Why this change? This doesn't build for me anymore.

Comment (docelic) [reply email.c:5]: Hey, the file is actually in include/urweb/urweb.h, and there was inconsistent use of including urweb.h vs urweb/urweb.h in the sources. Without this change, the file "urweb.h" alone was no longer found when the prefix was changed with ./configure --prefix=.
It is possible that I missed the other group of use cases which worked only as long as it wasn't changed.

**Comment by mdempsky** [comment]
This broke using --prefix=$HOME/opt/urweb for me though. configure.ac already accounts for the urweb subdirectory by including it in the include path argument.

File: email.c (hunk 1 above)
Comment (mdempsky) [reply email.c:5]: This broke using --prefix=$HOME/opt/urweb for me though. configure.ac already accounts for the urweb subdirectory by including it in the include path argument.

Comment (docelic) [reply email.c:5]: OK, I am just starting the compilation anew and will use a custom prefix to verify everything. Will report back to this thread.

Comment (docelic) [reply email.c:5]: Hey, I've checked and reproduced the issue you're reporting.
However, I believe that the change made is correct. You are using a non-standard prefix ($HOME/opt/urweb), so it would be expected that you invoke 'configure' with `./configure --prefix=$HOME/opt/urweb CFLAGS=-I$HOME/opt/urweb/include` to account for this.

An alternative and also good solution would be that all urweb's programs/modules add $PREFIX/include to the CFLAGS automatically, so nothing extra would be needed on behalf of the user.

This would be useful to decide on now since a similar pull request is pending for other modules too.

Thoughts/comments?

Comment (mdempsky) [reply email.c:5]: I've installed tons of packages (including urweb and urweb/email, before this CL) into $HOME/opt/foo and never had to set anything more than just --prefix. My understanding is that's the whole point of --prefix.

Comment (mdempsky) [reply email.c:5]: configure.ac sets:

https://github.com/urweb/email/blob/38f36a38f988019cdeb0982532b65c0ba0799eb4/configure.ac#L22

and then Makefile.am sets:

https://github.com/urweb/email/blob/38f36a38f988019cdeb0982532b65c0ba0799eb4/Makefile.am#L9

To change `#include <urweb.h>` to `#include <urweb/urweb.h>` then at least configure.ac should be updated accordingly.

Comment (docelic) [reply email.c:5]: Right, yes, I believe that it would be more correct for all urweb programs to add `$PREFIX/include` to CFLAGS rather than `$PREFIX/include/urweb` which they seem to have been adding so far.
